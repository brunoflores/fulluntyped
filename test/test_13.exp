{ ./test/test_13.f; 1; 0; 105 } field x not in record provided here


