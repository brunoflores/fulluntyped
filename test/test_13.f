let f = (lambda x:{x:Bool}. x.x) in
let record1 = {x = true, y = false} in
let record2 = {y = false} in 
f record2;
