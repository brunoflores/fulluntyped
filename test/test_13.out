
4 | f record2;
      ^

field x not in record provided here


